tests/hillel-auto.spec.js

```diff
@@ -8,14 +8,15 @@ test.describe("Base page test", () => {
     const randomIndex = Math.floor(Math.random() * 1000);
     const email = `testuser${randomIndex}@example.com`;
     await page.goto("https://guest:welcome2qauto@qauto.forstudy.space/");
```

Comment:
```suggestion
    await page.goto("https://qauto.forstudy.space/");
```
Ти заново намагаєшся авторизуватися (креди вказано перед "@"). Це зайве, так як в конфіг файлі вже додано httpCredentials